packages/jupyter-ai/jupyter_ai/extension.py

```diff
@@ -527,20 +531,54 @@ def _init_chat_handlers(self, ychat: YChat) -> Dict[str, BaseChatHandler]:
                     continue
 
             if command_name in chat_handlers:
-                self.log.error(
-                    f"Unable to register chat handler `{chat_handler.id}` because command `{command_name}` already has a handler"
+                self.log.warn(
+                    f"Overriding existing handler `{command_name}` with `{chat_handler.id}`."
                 )
```

Comment:
This doesn't seem necessary anymore because we are filtering out duplicates when getting the list of chat handler entry points (on line 457), correct?